src/pages/reporting/ReportTable.tsx

```diff
@@ -10,31 +10,68 @@ import {
   dateTemplate,
   permissionTemplate,
 } from 'components/table/templates';
+import classNames from 'classnames';
+import './ReportTable.scss';
+import {Link} from 'react-router-dom';
+import {createBrowserHistory} from 'history';
+
 
 export default () => {
   const type = getLastElementOfPath();
   const range = useSelector(selectRange);
   const report = useSelector(selectReport);
   const dispatch = useDispatch();
+
+  const history = createBrowserHistory();
+
   useEffect(() => {
     dispatch(getReport(range));
   }, [range]);
-  return (
-    <div className="mt-4">
-      <Table value={report[type].elements}>
-        <Column field="protectedEntity" header="Protected Entity" />
-        <Column field="policy" header="Policy" />
-        {type === 'backups' && (
-          <Column field="snapshotTime" header="Snapshot Time" />
-        )}
-        {type === 'restores' && (
-          <Column field="restoreTime" header="Restore Time" />
-        )}
-        <Column field="size" header="Size" body={sizeTemplate} />
-        <Column field="type" header="Type" />
-        <Column field="status" header="Status" />
-        <Column field="statusInfo" header="Status Info" />
-      </Table>
-    </div>
-  );
+
+  const statusVirtualEnvironment = (rowData) => {
```

Comment:
move all jsx functions outside this function and instead of returning reportsList() return template